docs/separate-benchmark-module.md

```diff
@@ -0,0 +1,102 @@
+# Creating Separate Module for Benchmarks
+
+This guide will walk you through the process of creating a separate module for benchmarks in your Kotlin project. 
+Creating a separate module for benchmarks is especially beneficial when you are integrating benchmarks into an existing project. 
+Here are a few advantages of doing so:
+
+1. **Organization**: It helps maintain a clean and organized project structure. Segregating benchmarks from the main code makes it easier to navigate and locate specific code segments.
+
+2. **Isolation**: Separating benchmarks ensures that the benchmarking code does not interfere with your main code or test code. This isolation guarantees accurate measurements without unintentional side effects.
+
+3. **Flexibility**: Creating a separate module allows you to manage your benchmarking code independently. You can compile, test, and run benchmarks without impacting your main source code.
```

Comment:
That's a fair point and I think the list should starts with it.

Reply:
Done